grafana-plugin/src/containers/IntegrationContainers/ExpandedIntegrationRouteDisplay/ExpandedIntegrationRouteDisplay.module.scss

```diff
@@ -47,3 +51,42 @@
 .default-route-view {
   min-height: 40px;
 }
+
+.block {
```

Comment:
Both `Expanded*` & `Collapsed*` components share some of the styles, but this will be further cleaned up when we migrate all `Integration` components to use emotion. Any additional refactoring to the existing *legacy* css/scss files is not worth it.